src/rf/apps/workers/emr.py

```diff
@@ -111,11 +140,10 @@ def add_steps(layer, status_queue, cluster_id):
     cluster_id - Existing EMR cluster ID
     """
     client = boto3.client('emr')
-    response = client.add_job_flow_steps(
+    client.add_job_flow_steps(
         JobFlowId=cluster_id,
         Steps=get_steps(layer, status_queue),
     )
-    log.info(response)
```

Comment:
Consider keeping this but use log.debug instead (same thing to other log.info statement)?